src/SprykerEco/Yves/Computop/Theme/default/direct_debit.twig

```diff
@@ -1,9 +1 @@
-<div class="small-12 medium-9 large-6">
-    <div class="small-12 columns">
-        {{ form_widget(form.computopDirectDebit.url) }}
-
-        <a href="{{ form.computopDirectDebit.vars.value.url }}">
-            {{ 'Pay with Computop Direct Debit' | trans }}
-        </a>
-    </div>
-</div>
\ No newline at end of file
+{{ form_widget(form.computopDirectDebit) }}
```

Comment:
This (and 3 others) are empty forms.
Why do you need those to be rendered ?

Reply:
Payment step works fine without this render but I think it was added to provide example where and how customers could customize computop.